src/components/ProvideDetails.jsx

```diff
@@ -158,11 +159,14 @@ const provideDetails = props => {
 	const goServiceRequestForm = (values) => {
 		GoHome(props);
 	};
+	const UpperCaseFirstLetter = (address, city, zip) => {
```

Comment:
based on the comment above, this should take in an array and handle any number of values. Would look something like this.



```js
String.prototype.capitalize = function() {
    return this.charAt(0).toUpperCase() + this.slice(1);
};

const toUpperCase = (strings) => strings.map(string => string.capitalize()).join(', ');
```

These both could be utilities and shouldn't be embedded in this component, so we can reuse.

It is also probably worth noting that this formatting logic should probably be handled by the web service, and if not the web service the service that provides the address. We want to limit the logic in these components as much as possible.